components/style/values/computed/length.rs

```diff
@@ -188,6 +188,13 @@ pub enum LengthOrPercentage {
     Calc(CalcLengthOrPercentage),
 }
 
+impl Default for LengthOrPercentage {
```

Comment:
And actually, what you want is a `Zero` trait. We already depend on something similar for euclid (`num_traits`?), so let's use that.